package.json

```diff
@@ -28,10 +31,19 @@
     "express": "^4.17.1",
     "express-fileupload": "^1.1.6-alpha.6",
     "html": "^1.0.0",
+    "ipld-dag-pb": "^0.18.2",
     "request": "^2.88.0",
     "request-promise-native": "^1.0.7",
     "serve-index": "^1.9.1",
     "solc": "^0.5.7",
```

Comment:
We should increase this to 0.6.0.

Reply:
Upgraded to ^0.6.1 in 754add8a7569cf7bcfdf1ec4d8600a63f51cac03